scheduler/src/cook/cors.clj

```diff
@@ -0,0 +1,59 @@
+(ns cook.cors)
+
+(defn preflight?
+  [req]
+  (= :options (:request-method req)))
+
+(defn same-origin?
+  "Returns true if the request is from the same origin as the provided origin header"
```

Comment:
Nitpick: the docstring should mention that it expects `origin` to be non-nil.

Reply:
I think `false` is actually a reasonable return value for this if origin is nil.